toolbelt/utils/converter.py

```diff
@@ -13,6 +13,7 @@
 def dockerhub2github_repo(dockerhub_repo: str):
     dockerhub2github_repo_map = {
         "ninechronicles-headless": HEADLESS_REPO,
+        "lib9c-stateservice": HEADLESS_REPO,
```

Comment:
It's not independent service?

Reply:
It is an independent service but it uses the same repository now.

Reply:
Ok, good